src/main/java/org/mule/extension/http/api/request/builder/QueryParam.java

```diff
@@ -16,18 +16,20 @@
  * 
  * @since 1.1
  */
-public class QueryParam {
+public class QueryParam implements KeyValuePair {
 
   @Parameter
   private String key;
```

Comment:
Shouldn't you specify that these shouldn't support expressions?

Reply:
Why? We are currently going to accept strings and that's it, but we would actually like it if the SDK+Studio give support for expressions in here

Reply:
This is going to be used as connectivity testing and sources, both being contexts in which expressions are not available. But most importantly, because this is going to be used for connectivity testing at the connection provider level, an expression here could mislead the SDK to generate a dynamic config unnecessarily, resulting in a performance degradation the user would have a hard time figuring out.